css/base.css

```diff
@@ -19,7 +19,7 @@ a {
 a:hover,
 a:focus {
   background-color: #e6eee0;
-  color: #cc2a00;
+  color: #444444;
```

Comment:
Hi Carrie,

This change make it look like the change on hover is now very subtle. When it was red/orange it looked much more dramatic.

Is this your intended effect?
Current hover:
![screen shot 2018-01-12 at 09 32 32](https://user-images.githubusercontent.com/2183332/34868397-8fa068a6-f77b-11e7-9f4f-286129bba15e.png)

Proposed hover:
![screen shot 2018-01-12 at 09 31 54](https://user-images.githubusercontent.com/2183332/34868408-9600af12-f77b-11e7-8591-56165ae590f0.png)

Reply:
Some people cannot read red/orange text. If you choose to stick with the pea colored background, then you are limited in your options for copy text color. Some dark blue colors could work.